translations/ui-users/en.json

```diff
@@ -827,5 +828,11 @@
   "accounts.actions.warning.waiveAction": "waived",
   "accounts.actions.warning.deselect": "Deselect to continue",
   "accounts.actions.cancellation.success": "<b>{count}</b> {count, plural, one {fee/fine} other {fees/fines}} for <b>{amount}</b> has been successfully <b>{action}</b> for <b>{user}</b>",
-  "accounts.status.outstanding": "Outstanding"
+  "accounts.status.outstanding": "Outstanding",
+  "transfers.label":"Fee/fine: Transfer accounts",
+  "transfers.columns.name":"Name*",
```

Comment:
Asterisks to label required fields should not be present in translations. They will be applied automatically by `stripes-components` form elements when given the prop `required`.